### PR TITLE
DEVX-664: remove multi datacenter client example to another repo where it is used

### PR DESCRIPTION
**Pull request by ybyzek**
Required for https://github.com/confluentinc/cp-docker-images/pull/676

### CODE REVIEW

File: clients/avro/src/main/java/io/confluent/examples/clients/basicavro/ConsumerMultiDatacenterExample.java [hunk 1]
```diff
@@ -17,29 +17,32 @@
...
-        props.put(ConsumerConfig.GROUP_ID_CONFIG, "java-consumer-app");
+        props.put(ConsumerConfig.GROUP_ID_CONFIG, "java-consumer-" + topic);
         props.put(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, "true");
         props.put(ConsumerConfig.AUTO_COMMIT_INTERVAL_MS_CONFIG, "1000");
         props.put(AbstractKafkaAvroSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG, schemaRegistryUrl);
         props.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
         props.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, KafkaAvroDeserializer.class);
-        props.put(ConsumerConfig.INTERCEPTOR_CLASSES_CONFIG, "io.confluent.connect.replicator.offsets.ConsumerTimestampsInterceptor");
+        props.put(ConsumerConfig.INTERCEPTOR_CLASSES_CONFIG, "io.confluent.connect.replicator.offsets.ConsumerTimestampsInterceptor,io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor");
+        props.put("confluent.monitoring.interceptor.bootstrap.servers", monitoringInterceptorBootstrapServers);
```
Comment (ghost) [inline clients/avro/src/main/java/io/confluent/examples/clients/basicavro/ConsumerMultiDatacenterExample.java]: Consider defining a constant for `confluent.monitoring.interceptor.bootstrap.servers` (not critical)

**Review by ghost — APPROVED** [review]
Other than the one minor comment about the literal property name `confluent.monitoring.interceptor.bootstrap.servers` (which isn't necessary to change), LGTM

File: clients/avro/src/main/java/io/confluent/examples/clients/basicavro/ConsumerMultiDatacenterExample.java (hunk 1 above)
Comment (ybyzek) [reply clients/avro/src/main/java/io/confluent/examples/clients/basicavro/ConsumerMultiDatacenterExample.java]: @mikebin : agree!  Done

**Review by gAmUssA — APPROVED** [review]
lgtm

**Pull request merged**